pkg/agent/cniserver/server_test.go

```diff
@@ -72,7 +72,7 @@ func TestLoadNetConfig(t *testing.T) {
 	cniService := newCNIServer(t)
 	networkCfg := generateNetworkConfiguration("testCfg", supportedCNIVersion)
 	requestMsg, containerID := newRequest(args, networkCfg, "", t)
-	netCfg, err := cniService.loadNetworkConfig(&requestMsg)
+	netCfg, err := cniService.loadNetworkConfig(requestMsg)
```

Comment:
I would suggest upgrading to a more recent version of Protobuf in a separate PR that we would merge first. These changes are a bit distracting when reviewing the PR.